ci/requirements/py37-min-all-deps.yml

```diff
@@ -8,42 +8,42 @@ dependencies:
   # When upgrading python, numpy, or pandas, must also change
   # doc/installing.rst and setup.py.
   - python=3.7
-  - boto3=1.9
-  - bottleneck=1.2
+  - boto3=1.12
+  - bottleneck=1.3
   - cartopy=0.17
   - cdms2=3.1
   - cfgrib=0.9
   - cftime=1.0
   - coveralls
-  - dask=2.9
-  - distributed=2.9
-  - h5netcdf=0.7
-  - h5py=2.9  # Policy allows for 2.10, but it's a conflict-fest
+  - dask=2.11
+  - distributed=2.11
+  - h5netcdf=0.8
+  - h5py=2.10  # Policy allows for 2.10, but it's a conflict-fest
```

Comment:
is that still the case? I can't really see any conflicts in CI but maybe that's because we use `mamba`?

Reply:
doesn't seem like that's still an issue. I removed the comment